routes/job/create.js

```diff
@@ -1,25 +1,52 @@
 import express from 'express';
 import { createJob } from '../../controllers/job-controller.js';
 import { createItem } from '../../controllers/stream-controller.js';
+import { StatusCodes } from 'http-status-codes';
+import { body } from 'express-validator/check/index.js';
+import validationHandler from '../../middlewares/validation-handler.js';
 
 const router = express.Router();
 
-router.post('/job', async (req, res) => {
-  const jobId = await createJob(req, res);
-  console.log('create job');
+router.post(
+  '/job',
+  [
+    body('company')
+      .isLength({ min: 3 })
+      .withMessage('Company must be at least 3 characters long'),
+    body('position')
+      .isLength({ min: 2 })
+      .withMessage('Position must be at least 2 characters long'),
+    body('status')
+      .isIn(['', 'interview', 'declined', 'pending'])
```

Comment:
status: interview, decline, pending, `offer`

Reply:
Good suggestion! Have been fixed it!